src/components/Media/MediaItem.tsx

```diff
@@ -31,7 +31,7 @@ const MediaItem: React.FC<Props> = ({
     >
       <figure>
         <div>
-          <img src={thumbnail.id} />
+          {thumbnail?.id && <img src={thumbnail.id} />}
```

Comment:
Love this.

Reply:
Yea, one-liner conditionals make life a little easier eh?